azure-pipelines.yml

```
@@ -1,24 +1,28 @@
-pr: none
+pr:
+  branches:
+    include:
+    - master
 
 trigger:
   branches:
     include:
-    - '*'
+    - master
```

```suggestion
    -'*'
```
I think we would want all PRs to create a review app, not only those created against `master`. This would allow us to have longer running feature branches.